src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -844,8 +873,25 @@ li {
     font-weight: normal;
   }
 
+  .v-input--checkbox .v-input__control .v-input__slot .v-label.theme--light {
+    padding-top: 2px;
```

Comment:
This is OK. (In my PR I moved the checkbox icon up instead but I prefer THIS fix.)

However, add a comment here saying what you're doing, eg, "// align text with checkbox"